docs/tutorial/using-checkbox/test-report.rst

```diff
@@ -0,0 +1,129 @@
+.. _test-report:
+
+Review Test Report 
+==================
+
+Once you finish your first test run, you may want to review the test reports. 
+Checkbox generates test reports in different formats that can be used to 
+easily share the results of a test session. By default, a :ref:`text summary<
```

Comment:
for some reason this doesn't work, put the link all on one line or it will not render (the `<` captures all chars, including the new line -> `WARNING: undefined label: ' text-summary'`)

Reply:
fixed in [6159a36](https://github.com/canonical/checkbox/pull/637/commits/6159a3674bc44d689c7a6ab7acda85d5f68d13d1).